pkg/helm/helm_cli.go

```diff
@@ -450,19 +430,6 @@ func (h *HelmCLI) Template(chart string, releaseName string, ns string, outDir s
 // UpgradeChart upgrades a helm chart according with given helm flags
 func (h *HelmCLI) UpgradeChart(chart string, releaseName string, ns string, version string, install bool, timeout int, force bool, wait bool, values []string, valueStrings []string, valueFiles []string, repo string, username string, password string) error {
 	var err error
-	currentNamespace := ""
-	if h.Binary == "helm3" {
```

Comment:
I like seeing the removal of these comparisons against "helm3". This looks promising. What issue/behaviour do you try to fix/implement with this change? Could you capture it in an issue?

Reply:
in helm3 alpha we used to switch the namespace manually in the kubeconfig file as the flag "--namespace" wasn't working, now that we use a later version in our container, there is no more need to modify the config. this basically juste a cleanup... not a real issue